### PR TITLE
add html stub for dat 404 fixes #190

### LINKED ISSUE #190: UX: message to user when archive isn't available (or doesnt exist)
see:
https://github.com/datproject/dat.land-roadmap/issues/47

### PR DESCRIPTION
**Pull request by karissa**
@Kriesse this needs some css love

### CODE REVIEW

**Comment by karissa** [comment]
see https://github.com/datproject/dat.land-roadmap/issues/47

**Comment by karissa** [comment]
closes #190

File: client/js/pages/archive/index.js [hunk 1]
```diff
@@ -9,7 +9,24 @@ const prettyBytes = require('pretty-bytes')
...
+            <img src="./public/img/dat-data-logo.svg" />
+          </div>
+          <h3>404 - Sorry, no dat here.</h3>
+          <p class="mb3">We couldn't find the droids you were looking for. Is the link correct?</p>
+          <p class="mb4">
+            <button class="btn btn--large btn--green take-me-home" href="/">Take me home</button>
+          </p>
+        </div>
+      </div>`
+  }
```
Comment (laurengarcia) [inline client/js/pages/archive/index.js]: seems like this block ^^ should be abstracted away so it can be shared with non-archive routes?

Comment (karissa) [inline client/js/pages/archive/index.js]: yes that's right. i'll make a 404 template

**Pull request merged**